index.html

```diff
@@ -1,206 +1,277 @@
-
-        <section class="about-section">
-            <h1 class="about-heading">About</h1>
-            <p class="about">Lorem ipsum dolor sit amet, consectetur 
-                adipiscing elit, sed do eiusmod tempor 
-                incididunt ut labore et dolore magna aliqua. 
-                Ut enim ad minim veniam, quis nostrud exercitation 
-                ullamco laboris nisi ut aliquip ea eiusmod
-            </p>
-            <button class="cv">Get my CV <i class="fa fa-download"></i></button>
+    <section class="skills-section">
+        <section class="languages">
+            <h1 class="languages-heading">Languages</h1>
+            <ul>
+                <li>Javascript(ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
         </section>
+        <section class="frameworks">
+            <h1 class="frameworks-heading">Frameworks</h1>
+            <ul>
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <li>Selenium</li>
+            </ul>
+        </section>
+        <section class="skills">
+            <h1 class="skills-heading">Skills</h1>
+            <ul>
+                <li>Codekit</li>
+                <li>Codepen</li>
+                <li>Github</li>
+                <li>GitLAb</li>
+                <li>Terminal</li>
+            </ul>
+        </section>
+    </section>
 
-        <!--Languages, frameworks, and skills-->
-
-        <section class="skills-section">
-            <section class="languages">
-                <h1 class="languages-heading">Languages</h1>
-                <ul>
-                    <li>Javascript(ES5/ES6)</li>
-                    <li>Ruby</li>
-                    <li>HTML</li>
-                    <li>CSS</li>
-                </ul>
-            </section>
-            <section class="frameworks">
-                <h1 class="frameworks-heading">Frameworks</h1>
-                <ul>
-                    <li>Bootstrap</li>
-                    <li>Ruby on rails</li>
-                    <li>RSpec</li>
-                    <li>Capibara</li>
-                    <li>Selenium</li>
-                </ul>
-            </section>
-            <section class="skills">
-                <h1 class="skills-heading">Skills</h1>
-                <ul>
-                    <li>Codekit</li>
-                    <li>Codepen</li>
-                    <li>Github</li>
-                    <li>GitLAb</li>
-                    <li>Terminal</li>
-                </ul>
-            </section>
+    <!--Form-->
+
+    <section class="contact-form">
+        <img alt="Logo" class="geometry" src="./images/image_geometry_1.png">
+        <h2 class="contact-heading">Get started</h2>
+        <p class="contact-paragraph">Lorem ipsum dolor sit amet,
+            consectetur adipiscing elit,
+            sed do eiusmod tempor incididunt
```

Comment:
- [ ] Good job on adding contact form section but kindly wrap it inside of footer as it was part of [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Keep rocking 💪